WalletWasabi/JsonConverters/ExtPubKeyJsonConverterNg.cs

```diff
@@ -0,0 +1,36 @@
+using NBitcoin;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using WalletWasabi.Helpers;
+
+namespace WalletWasabi.JsonConverters;
+
+public class ExtPubKeyJsonConverterNg : JsonConverter<ExtPubKey>
+{
+	/// <inheritdoc />
+	public override ExtPubKey? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+	{
+		if (reader.TokenType != JsonTokenType.String)
+		{
+			throw new JsonException("Expected a JSON string value.");
```

Comment:
Here the converter looks like it handles `null`, but at the same time `null` throws.

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/11840/commits/69c166ccdab92dccbc03e79b76fd419b3f8c0761

Added test for `null` and `"null"`, both handles both the same way.